packages/block-library/src/navigation/index.php

```diff
@@ -449,9 +449,16 @@ function render_block_core_navigation( $attributes, $content, $block ) {
 		}
 
 		$inner_blocks = new WP_Block_List( $fallback_blocks, $attributes );
-
 	}
 
+	/**
+	 * Filter navigation block $inner_blocks.
+	 * Allows modification of a navigation block menu items.
+	 *
+	 * @param \WP_Block_List $inner_blocks
+	 */
+	$inner_blocks = apply_filters( 'render_block_core_navigation_inner_blocks', $inner_blocks );
```

Comment:
There is one filter that uses a different naming pattern. Let's align it with:

https://github.com/WordPress/gutenberg/blob/8247dd0599ccbd63b332238f9caa000742723528/packages/block-library/src/navigation/index.php#L333-L345